assets/css/js_interop.css

```diff
@@ -296,20 +296,58 @@ solely client-side operations.
 
 /* === Session views === */
 
-[data-el-session][data-js-view="code-zen"] [data-el-view-toggle="code-zen"] {
+[data-el-session][data-js-view] [data-el-view-activate] {
+  @apply pointer-events-none;
+}
+
+[data-el-session]:not([data-js-view]) [data-el-view-deactivate-button] {
+  @apply hidden;
+}
+
+[data-el-session][data-js-view="output-panel-popped-out"]
+  [data-el-view-activate="output-panel"],
+[data-el-session]:not([data-js-view="output-panel-popped-out"])
+  [data-el-view-output-panel-popin-button] {
+  @apply hidden;
+}
+
+[data-el-session][data-js-view="output-panel"]
+  [data-el-view-activate="output-panel"],
+[data-el-session][data-js-view="output-panel-popped-out"]
+  [data-el-view-output-panel-popin-button] {
+  @apply text-green-bright-400;
+}
+
+[data-el-session][data-js-view="code-zen"] [data-el-view-activate="code-zen"] {
   @apply text-green-bright-400;
 }
 
 [data-el-session][data-js-view="presentation"]
-  [data-el-view-toggle="presentation"] {
+  [data-el-view-activate="presentation"] {
   @apply text-green-bright-400;
 }
 
-[data-el-session][data-js-view="custom"] [data-el-view-toggle="custom"] {
+[data-el-session][data-js-view="custom"] [data-el-view-activate="custom"] {
   @apply text-green-bright-400;
 }
 
-[data-el-session][data-js-view]
+[data-el-session]:not([data-js-view="output-panel"]) [data-el-output-panel] {
+  @apply hidden;
+}
+
+[data-el-session][data-js-view="output-panel"] [data-el-notebook-content] {
+  @apply absolute w-1/2 left-0 px-16 py-4 overflow-auto;
```

Comment:
tried to make notebook and output panel independent scrollable. Couldn't figure out how to do it yet.
Is this approach to have to panels side by side correct?
```html
<div class="absolute w-1/2 left-0 px-16 py-4 overflow-auto" data-el-notebook-content>
NOTEBOOK
</div>
<div class="absolute w-1/2 right-0 h-full overflow-auto" data-el-output-panel>
OUTPUT PANEL
</div>
```

Reply:
You can probably have the output panel side by side with `data-el-notebook`, so one level up in the DOM?